src/celeritas/phys/InteractionUtils.hh

```diff
@@ -58,7 +58,8 @@ struct ExitingDirectionSampler
     template<class Engine>
     inline CELER_FUNCTION Real3 operator()(Engine& rng)
     {
-        UniformRealDistribution<real_type> sample_phi(0, 2 * constants::pi);
+        UniformRealDistribution<real_type> sample_phi(
+            0, real_type(2 * constants::pi));
```

Comment:
I think that would work with `()` but not `{}`, because the `2.0 * pi` is a double expression, which can't be implicitly converted to a single precision under C++11 rules.

What I was actually thinking, since we have so many `sample_phi` in our code, is a free function `sample_twopi(RNG&)` (or `sample_azi` or something) that we could just call.